src/utils/fuse.ts

```diff
@@ -0,0 +1,187 @@
+// @ref: https://github.com/ensdomains/ensjs-v3/blob/feat/namewrapper-upgrade/packages/ensjs/src/utils/fuses.ts
+
+// child named fuses
+const CANNOT_UNWRAP = 1;
+const CANNOT_BURN_FUSES = 2;
+const CANNOT_TRANSFER = 4;
+const CANNOT_SET_RESOLVER = 8;
+const CANNOT_SET_TTL = 16;
+const CANNOT_CREATE_SUBDOMAIN = 32;
+const CANNOT_APPROVE = 64;
+
+// parent named fuses
+const PARENT_CANNOT_CONTROL = 0x10000;
+const IS_DOT_ETH = 0x20000;
+const CAN_EXTEND_EXPIRY = 0x40000;
+
+// fuse ranges
+export const CHILD_CONTROLLED_FUSES = 0x0000ffff;
+export const PARENT_CONTROLLED_FUSES = 0xffff0000;
+export const USER_SETTABLE_FUSES = 0xfffdffff;
+
+// empty fuse
+const CAN_DO_EVERYTHING = 0;
+
+export const childFuseEnum = {
+  CANNOT_UNWRAP,
+  CANNOT_BURN_FUSES,
+  CANNOT_TRANSFER,
+  CANNOT_SET_RESOLVER,
+  CANNOT_SET_TTL,
+  CANNOT_CREATE_SUBDOMAIN,
+  CANNOT_APPROVE,
+} as const;
+
+export const parentFuseEnum = {
+  PARENT_CANNOT_CONTROL,
+  CAN_EXTEND_EXPIRY,
+};
+
+export const fullParentFuseEnum = {
+  ...parentFuseEnum,
+  IS_DOT_ETH,
+} as const;
+
+export const userSettableFuseEnum = {
+  ...childFuseEnum,
+  ...parentFuseEnum,
+} as const;
+
+export const fullFuseEnum = {
+  ...userSettableFuseEnum,
+  CAN_DO_EVERYTHING,
+};
+
+export const unnamedChildFuses = [
+  64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768,
```

Comment:
64 should not be an unnamed child fuse anymore, as that is now the `CANNOT_APPROVE` fuse

Reply:
ah, great catch!